libqtile/core/session_manager.py

```diff
@@ -1,14 +1,24 @@
 import asyncio
 import os
+import tempfile
+from typing import Optional
 
 from libqtile import ipc
 from libqtile.backend import base
+from libqtile.core.lifecycle import lifecycle
+from libqtile.core.loop import QtileLoop
 from libqtile.core.manager import Qtile
 
 
 class SessionManager:
     def __init__(
-        self, kore: base.Core, config, *, fname: str = None, no_spawn=False, state=None
+        self,
+        kore: base.Core,
+        config,
+        *,
+        fname: str = None,
+        no_spawn: bool = False,
+        state: bool = None,
```

Comment:
Please don't change code if not necessary, it clutters your commit and makes the history harder to navigate.